components/ExerciseDisplayed/index.js

```diff
@@ -19,9 +19,9 @@ export default function ExerciseDisplayed({
           onClick={() => {
             handleFavoriteClick(dat);
           }}
-          className="small-border bookmark"
-          src="/bookmarkstar.svg"
-          alt="star image"
+          className="bookmark"
```

Comment:
You shouldn't need a `className` when using styled components. Can't you incorporate these styles into `StyledImage`?